.github/workflows/deploy.yml

```
@@ -0,0 +1,61 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ 'develop', 'master' ]
+  pull_request:
+    branches: [ 'develop' ]
```

Why not also `master` here?
```suggestion
    branches: [ 'develop' , 'master' ]
```